templates/dispatch.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: dispatch
+  labels:
+    kubeaddons.mesosphere.io/name: dispatch
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.3.0"
+    appversion.kubeaddons.mesosphere.io/dispatch: "0.3.0"
+    endpoint.kubeaddons.mesosphere.io/dispatch: /dispatch/tekton/
```

Comment:
I was wondering if we should have this if we create our own Ingress records?

Reply:
This annotation is read by the konvoy/kommander UI and used to create the link for the punchout card.

If or when we have docs you would also want to add a link to those with:
`docs.kubeaddons.mesosphere.io/dispatch: "<URL_TO_DOCS>"`

Reply:
this should be `/tekton:` so that the punch out is labels Tekton and not Dispatch.

also make sure the appversion matches the version you want to display in the UI for the link to Tekton.

Reply:
It sounds like we're calling tekton dispatch for now? https://github.com/mesosphere/kubeaddons-configs/pull/328#issuecomment-554516441

Reply:
So I was thinking we would call it `dispatch` and `argo-cd` for now, since tekton is more of an implementation detail and eventually we'll have the Tekton UI fully wrapped (possibly deep link to it from the "Dispatch UI"). So I don't think we need a Tekton badge - we can use this as the entry point / logo / docs link for Dispatch.